src/httprpc.cpp

```diff
@@ -351,7 +350,8 @@ static bool HTTPReq_JSONRPC(HTTPRequest* req, const std::string &)
 
 static bool InitRPCAuthentication()
 {
-    if (gArgs.GetArg("-rpcpassword", "") == "")
+    // SYSCOIN
+    if (gArgs.GetArg("-rpcpassword", "") == "p")
```

Comment:
might still need to check for ""
if (gArgs.GetArg("-rpcpassword", "") == "" || gArgs.GetArg("-rpcpassword", "") == "p")